### ISSUE TITLE
[DOC] Examples of using NVT tags and selectors

### ISSUE THREAD

**Issue by karlhigley** [NVIDIA-Merlin/NVTabular#1296]
**Is your feature request related to a problem? Please describe.**
We now have the ability to select columns from schemas using tags while constructing NVT workflows, but don't yet have documentation of how to use this feature.

**Describe the solution you'd like**
Create a new example that explains how to use schemas and tags.

**Describe alternatives you've considered**
- Integrate tags into the existing examples of how to construct workflows

**Additional context**
This functionality was created for integration with Transformers4Rec, which I think does have some examples.

**Comment by bschifferer** [comment]
I dont think we need an extra example for Column Tagging API. I integrated the feature in the scaling criteo example.
